common/main/en.xml

```diff
@@ -3462,15 +3462,6 @@ annotations.
 			<zone type="Etc/Unknown">
 				<exemplarCity>Unknown City</exemplarCity>
 			</zone>
-			<zone type="Antarctica/DumontDUrville">
-				<exemplarCity>Dumont d’Urville</exemplarCity>
```

Comment:
Why?

Reply:
en.xml doesn't need to be 'complete', except for sideways inheritance. We require other locales to be complete so that we are sure that they have considered each data item.